### PR TITLE
Configure vm.swappiness

### PR DESCRIPTION
**Pull request by manics**
Attempt to reduce swap load http://serverfault.com/a/692515

This role is incomplete and still needs further work to set the swap partition size. Think of it more as documentation for a planned live change to narwhal and a requirement for when the server is re-imaged.

### CODE REVIEW

File: ansible/roles/server-swap/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+# defaults file for roles/server-swap
+
+sysctl_vm_swappiness: 1
```
Comment (kennethgillen) [inline ansible/roles/server-swap/defaults/main.yml]: Kernel version 3.5 and over, as well as kernel version 2.6.32-303 and over: Minimum amount of swapping without disabling it entirely.

**Comment by kennethgillen** [comment]
:+1: Not run the ansible, but the config change looks sane, and worth a try.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [INFRASTRUCTURE-merge#43](https://ci.openmicroscopy.org/job/INFRASTRUCTURE-merge/43/). See the [console output](https://ci.openmicroscopy.org/job/INFRASTRUCTURE-merge/43/consoleText) for more details.
Possible conflicts:
  - PR #38 manics 'systemctl daemon-reload, other docker fixes'
    - ansible/idr-provision.yml

**Comment by manics** [comment]
Commits copied to #38 to avoid conflict

**Pull request closed without merging**